dev/drivers/scripts/cam/stats/jevs_cam_hireswfv3_precip_stats.sh

```diff
@@ -51,7 +51,8 @@ export MET_CONFIG="${MET_PLUS_PATH}/parm/met_config"
 export PYTHONPATH=$HOMEevs/ush/$COMPONENT:$PYTHONPATH
 
 # Developer Settings
-export DATAROOT=/lfs/h2/emc/stmp/$USER/evs_test/$envir/tmp
+export envir=prod
+export DATAROOT=/lfs/h2/emc/stmp/$USER/evs_test/$venvir/tmp
 export COMROOT=/lfs/h2/emc/vpppg/noscrub/${USER}
```

Comment:
@ShelleyMelchior-NOAA 

You seem to have introduced some sort of typo here on line 55 in green.

Reply:
corrected